src/annotator/guest.js

```diff
@@ -491,34 +552,59 @@ export default class Guest extends Delegator {
     const targets = Promise.all([info, selectors]).then(setTargets);
 
     targets.then(() => this.publish('beforeAnnotationCreated', [annotation]));
-    targets.then(() => this.anchor(annotation));
+    targets.then(() => this.anchor(/** @type {AnnotationData} */ (annotation)));
 
     if (!annotation.$highlight) {
       this.crossframe?.call('showSidebar');
     }
     return annotation;
   }
 
+  /**
+   * Create a new annotation with the `$highlight` flag set.
+   *
+   * This flag indicates that the sidebar should save the new annotation
+   * immediately rather than prompting the user to write a comment.
+   */
   createHighlight() {
     return this.createAnnotation({ $highlight: true });
   }
 
+  /**
+   * Show annotations in the sidebar.
+   *
+   * @param {AnnotationData[]} annotations
+   */
   showAnnotations(annotations) {
     const tags = annotations.map(a => a.$tag);
     this.crossframe?.call('showAnnotations', tags);
     this.crossframe?.call('showSidebar');
   }
 
+  /**
+   * @param {AnnotationData[]} annotations
```

Comment:
This is a method that I feel could use a comment :) toggle what? (OK, yes, I know the answer, but...)